mobile/src/calendar/event-dialog/edit-dayoff-event-dialog.tsx

```diff
@@ -34,10 +36,22 @@ class EditDayoffEventDialogImpl extends Component<EditDayoffEventDialogProps & E
             onClosePress={this.onCloseClick}/>;
     }
 
+    //----------------------------------------------------------------------------
+    private getDialogTypeTitle = (): string => {
```

Comment:
Consider renaming it to 'getDialogTitle' or just 'getTitle'

Reply:
getEventTitle

Reply:
c4fb50a7c9c27cd61d5a6c21ae82b6d107222a29